kernelci.org

```diff
@@ -197,17 +197,20 @@ cmd_docker() {
     cd checkout/kernelci-core/config/docker
 
     core_rev=$(git show --pretty=format:%H -s origin/kernelci.org)
-    args="build --push --no-cache --build-arg core_rev=$core_rev"
+    base_args="build --build-arg core_rev=$core_rev"
+    args="$base_args --push --no-cache"
 
     # KernelCI tools
     ./kci_docker $args kernelci
     ./kci_docker $args k8s --fragment=kernelci
 
     # Compiler toolchains
     for clang in clang-11 clang-12 clang-13 clang-14 clang-15 clang-16; do
-	./kci_docker $args $clang --fragment=kselftest --fragment=kernelci
-        for arch in arm arm64 mips riscv64 x86; do
-	    ./kci_docker $args $clang --arch $arch \
+	./kci_docker $args $clang
+    done
+    for clang in clang-11 clang-15 clang-16; do
```

Comment:
Just curiosity, we build arch specific images just for 11, 15 ,16?

Reply:
Sorry, just got confused by patches, ignore this question

Reply:
Yes because these are the only ones used in production.